### ISSUE TITLE
Asynchronous Content on IE

### ISSUE THREAD

**Issue by SFPink** [jlmakes/scrollreveal.js#233]
Hello,

I have implemented the async method, works great on chrome as you would expect. However on IE the tiles do not get set to visible. IS there something I've missed?

Have a look at my dev site - http://89.200.141.33/~nmattersdev/category/health-handover/

On IE 10 / 11 the console outputs this:

```
ScrollReveal is not supported in this browser.
sync() failed: no reveals found. (Despite the new content is loaded hidden)
```

**Comment by jlmakes** [comment]
@SFPink I changed the CSS Transition and CSS Transform support detection in `v3.0.7` (right here in [this commit](https://github.com/jlmakes/scrollreveal.js/commit/8a62a21c396ebb74a17bd08f7243a44d63198555)) and it would seem the approach does not suit Internet Explorer 10, 11, and likely Edge.

Can I provide you with a p

**Comment by SFPink** [comment]
I have just tested v3.0.6 and confirm that is does not fix the issue. Tiles seem to fade in and then back out when initially loaded. When the ajax method is called and the new tiles load in, they appear briefly then are hidden again.

I can leave this version on my dev site so you can see what I mean: http://89.200.141.33/~nmattersdev/category/health-handover/


I also tested version v3.0.9 out of curiosity as I was running v3.0.8, there appears to be a mistake with the inline css that gets added during transition it adds this to each element: "transition: visibility: visible" this only appears on IE.

**Comment by jlmakes** [comment]
>I also tested version` v3.0.9` out of curiosity as I was running `v3.0.8`, there appears to be a mistake with the inline css that gets added during transition it adds this to each element: "transition: visibility: visible" this only appears on IE.

Was this bug in all 3 versions you tested?

**Comment by SFPink** [comment]
It only happened in v3.0.9

**Comment by jlmakes** [comment]
I don’t believe this was actually fixed, since your tests with `v3.0.6` still failed.

**Comment by jlmakes** [comment]
I’ve tested the 3.1 release candidate, and it correctly checks support in IE9, 10, 11 and Edge.
